data/mm/world/ancient_castle_of_ikana.yml

```diff
@@ -1,40 +1,40 @@
 "Ancient Castle of Ikana Exterior":
-  region: IKANA_CASTLE
+  dungeon: IkanaCastle
```

Comment:
Please keep the name of the dungeons short (few characters max).
For the castle I suggest either `ACoI` or `AC`